### DISCUSSION TITLE
ASUS ROG Strix x670e-F support?

### DISCUSSION THREAD

**Discussion by TeNou7** [Rem0o/FanControl.Releases#1164] [General]
Hello, I am wondering if I did something wrong or if the new x670e or my board (title) in general are not supported yet. Will there be support soon? I see the GPU fans, but thats it. No normal case fans that are connected to the Mainboard.

Thank you!

- Zeno

**Comment by Rem0o** [comment]
Those boards are brand new. No support yet is likely. Will have to be added into:
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor

**Reply by TeNou7** [reply]
thanks for the fast reply and all the work!
